From 811c7b5a3efc9740151e0476844e5c8c60c690e0 Mon Sep 17 00:00:00 2001
From: Lukas Bulwahn <lukas.bulwahn@gmail.com>
Date: Sat, 21 Apr 2018 13:13:03 +0200
Subject: [PATCH] moveit_ros/perception: do not use SYSTEM

---
 moveit_ros/perception/CMakeLists.txt | 2 --
 1 file changed, 2 deletions(-)

diff --git a/moveit_ros/perception/CMakeLists.txt b/moveit_ros/perception/CMakeLists.txt
index dcd4436..c60b78e 100644
--- a/moveit_ros/perception/CMakeLists.txt
+++ b/moveit_ros/perception/CMakeLists.txt
@@ -84,8 +84,6 @@ include_directories(lazy_free_space_updater/include
                     ${perception_GL_INCLUDE_DIRS}
                     ${Boost_INCLUDE_DIRS}
                     ${catkin_INCLUDE_DIRS}
-                    )
-include_directories(SYSTEM
                     ${OpenCV_INCLUDE_DIRS}
                     ${EIGEN3_INCLUDE_DIRS}
                     ${SYSTEM_GL_INCLUDE_DIR}
-- 
2.7.4

